subprojects/docs/src/snippets/testing/test-suite-multi-configure-each-extracted/kotlin/build.gradle.kts

```
@@ -0,0 +1,68 @@
+import org.gradle.api.internal.tasks.testing.junit.JUnitTestFramework
+
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    `java-library`
+}
+
+version = "1.0.0"
+group = "org.gradle.sample"
+
+repositories {
+    mavenCentral()
+}
+
+// tag::multi-configure[]
+testing {
+    suites {
+        val applyMockito = { suite: JvmTestSuite -> // <1>
+            suite.useJUnitJupiter()
+            suite.dependencies {
+                implementation("org.mockito:mockito-junit-jupiter:4.6.1")
+            }
+        }
+
+        val test by getting(JvmTestSuite::class) {
+            applyMockito(this) // <2>
+        }
+
+        val integrationTest by registering(JvmTestSuite::class)
+        applyMockito(integrationTest.get()) // <3>
```

This makes it more similar to Groovy:
```suggestion
        val test by getting(JvmTestSuite::class, applyMockito)  // <2>

        val integrationTest by registering(JvmTestSuite::class, applyMockito) // <3>
```

We also don't want to encourage people to call `.get()` on items in the container.